test/langtools/tools/javac/T8297602.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug 8297602
  * @summary Compiler crash with type annotation and generic record during pattern matching
- * @compile -XDrawDiagnostics T8297602.java
+ * @compile/fail/ref=T8297602.out -XDrawDiagnostics T8297602.java
```

Comment:
I think that this test can be eliminated as new test AnnotationErrors supersedes it.